components/code-template.tsx

```diff
@@ -71,22 +75,33 @@ export default ({
 
   return (
     <Template title={title}>
-      <Block spaceInset='spaceInsetStretch020'>
+      <Block spaceInset="spaceInsetStretch020">
         <Stack flow="horizontal-center" stackDistribution="space-between">
-          <Block spaceInset='spaceInsetStretch020'>
+          <Block spaceInset="spaceInsetStretch020">
             <Heading2>{title}</Heading2>
             <P>{description}</P>
           </Block>
           {showThemeSwitcher && themeSwitcher}
-          {nextPage ? (
-            <Tag href={`/${nextPage}`} size="large">
-              Next Step 👉
-            </Tag>
-          ) : (
-            <Tag href="/" size="large">
-              Home 🏡
-            </Tag>
-          )}
+          <Block>
+            {prevPage && (
+              <Tag
```

Comment:
Can you please add next-link here so transitions between pages is smooth,  something like 
```
<Link href=""><Tag ... /></Link>
```

Reply:
Updated